adm/style/ppde_transactions.html

```diff
@@ -1,11 +1,112 @@
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_LAST_NAME') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="medium" id="last_name" name="last_name" value="{{ MT_LAST_NAME }}"></dd>
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_PAYER_EMAIL') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="medium" id="payer_email" name="payer_email" value="{{ MT_PAYER_EMAIL }}"></dd>
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_RESIDENCE_COUNTRY') ~ lang('COLON') }}</label><br><span>{{ lang('PPDE_MT_RESIDENCE_COUNTRY_EXPLAIN') }}</span></dt>
+			<dd><input type="text" class="tiny" id="residence_country" name="residence_country" value="{{ MT_RESIDENCE_COUNTRY }}" maxlength="2"></dd>
+		</dl>
+		<hr>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_TOTAL_AMOUNT') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label></dt>
+			<dd>
+				<input type="text" class="narrow" id="mc_gross" name="mc_gross" value="{{ MT_MC_GROSS }}">
+				<select id="mc_currency" name="mc_currency">
+					{% for options in loops.options %}
+					<option value="{{ options.CURRENCY_ISO_CODE }}"{% if (MT_MC_CURRENCY == options.CURRENCY_ISO_CODE) or (not MT_MC_CURRENCY and options.S_CURRENCY_DEFAULT) %} selected{% endif %}>{{ options.CURRENCY_ISO_CODE }}</option>
+					{% endfor %}
+				</select>
+			</dd>
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_FEE_AMOUNT') ~ lang('COLON') }}</label></dt>
+			<dd><input type="text" class="narrow" id="mc_fee" name="mc_fee" value="{{ MT_MC_FEE }}"> <span id="mc_fee_currency"></span></dd>
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_NET_AMOUNT') ~ lang('COLON') }}</label></dt>
+			<dd><span id="net_amount">0</span> <span id="net_amount_currency"></span></dd>
+		</dl>
+		<hr>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_PAYMENT_DATE') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label></dt>
+			<dd>
+				<span class="date-fields-container">
+					<span>{{ lang('YEAR') ~ lang('COLON') }}</span> <input type="text" class="tiny" id="payment_date_year" name="payment_date_year" value="{{ MT_PAYMENT_DATE_YEAR }}">
+					<span>{{ lang('MONTH') ~ lang('COLON') }}</span> <input type="text" class="tiny" id="payment_date_month" name="payment_date_month" value="{{ MT_PAYMENT_DATE_MONTH }}">
+					<span>{{ lang('DAY') ~ lang('COLON') }}</span> <input type="text" class="tiny" id="payment_date_day" name="payment_date_day" value="{{ MT_PAYMENT_DATE_DAY }}">
+				</span>
+				<span class="date-calendar-container" data-language="{{ lang('PPDE_MT_DATEPICKER_LANG') }}" title="{{ lang('PPDE_MT_PAYMENT_DATE_PICK') }}">
+					<i class="icon fa fa-fw fa-calendar" aria-hidden="true"></i><span class="sr-only">{{ lang('PPDE_MT_PAYMENT_DATE_PICK') }}</span> <span class="date-calendar-value"></span>
+				</span>
+			</dd>
+		</dl>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_PAYMENT_TIME') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label></dt>
```

Comment:
Should we use PHP's date format?

```
16:25:50 (H:i:s)
16:25 (G:i)
04:25:50 PM (h:i:s a)
4:25 PM (g:i a)
```

Reply:
Not sure to understand your question.
Do you want to use the PHP date format to provide an example in Explain or add as is the code mentioned below in Explain?

Reply:
I mean using the PHP date() format in the EXPLAIN text, instead of (hh:mm), like this:

![donation_time](https://user-images.githubusercontent.com/6710357/49703616-4fe24680-fc07-11e8-87b8-a8e05f8256bc.png)

Reply:
IMO, PHP format is not understandable for standard users. Time result should be enough.
```
16:25:50
16:25
04:25:50 PM
4:25 PM
```
For the explain text:
`Valid time formats`, `Time formats allowed` or `Time formats accepted`?

Reply:
The time used in the examples is now the current user time. The example formats are generated in PHP and are assigned to the template, where they are displayed all in one line. When printed as list, they used too much space and pushed down the memo field.